Dockerfile

```diff
@@ -3,3 +3,6 @@ FROM nginx:latest
 LABEL org.opencontainers.image.authors="pmc@voxpupuli.org"
 
 COPY . /usr/share/nginx/html/
+COPY entrypoint.sh /
```

Comment:
I'd move this above the other `COPY`. This file is less likely to change and you can use caching more efficiently. And if this file changes, the above will also be invalidated so you'd trash the cache anyway.